pkg/mcs/resource_manager/server/manager.go

```diff
@@ -121,6 +124,23 @@ func (m *Manager) Init(ctx context.Context) {
 		}
 	}
 	m.storage.LoadResourceGroupStates(tokenHandler)
+
+	// add default group
+	if _, ok := m.groups[defaultGroupName]; !ok {
+		m.groups[defaultGroupName] = &ResourceGroup{
```

Comment:
So default resource group is in memory only?

Reply:
It will persists periodically.